virttest/utils_misc.py

```diff
@@ -2818,199 +2818,322 @@ def get_uptime(session=None):
     return float(uptime.split()[0])
 
 
-def list_linux_guest_disks(session, partition=False):
+def get_linux_disks(session, partition=False):
```

Comment:
And to make these utilities easy to use, I'd suggest to implement them in this way:

1. introduce the following low-level utilities
```python
# create partition table
def create_partition_table_linux()
def create_partition_table_windows()
def create_partition_table()
# create single partition
def create_partition_linux()
def create_partition_windows()
def create_partition()
# create partitions (atomically)
def create_partition_batch_linux()
def create_partition_batch_windows()
def create_partition_batch()
# delete single partition
def delete_partition_linux()
def delete_partition_windows()
def delete_partition()
# delete partitions (atomically)
def delete_partition_batch_linux()
def delete_partition_batch_windows()
def delete_partition_batch()
# clean all the partitions
def clean_partitions_linux()
def clean_partitions_windows()
def clean_partitions()
# create file system
def create_filesyetem_linux()
def create_filesystem_windows()
def create_filesystem()
# linux only
def mount()
def umount()
# windows only
def set_drive_letter()
def drop_drive_letter()
```
2. introduce high-level utilities by combining up the basic ones for easier use